meta-oe/recipes-devtools/nlohmann-json/nlohmann-json_git.bb

```diff
@@ -2,13 +2,13 @@ SUMMARY = "JSON for modern C++"
 HOMEPAGE = "https://nlohmann.github.io/json/"
 SECTION = "libs"
 LICENSE = "MIT"
-LIC_FILES_CHKSUM = "file://LICENSE.MIT;md5=9a8ae1c2d606c432a2aa2e2de15be22a"
+LIC_FILES_CHKSUM = "file://LICENSE.MIT;md5=f5f7c71504da070bcf4f090205ce1080"
```

Comment:
what caused the change in checksum here ?

Reply:
This

> -Copyright (c) 2013-2018 Niels Lohmann
> +Copyright (c) 2013-2019 Niels Lohmann

Reply:
usually this should be documented in commit msg

Reply:
thanks, I will do it next time.